scanner/indexer/indexer.go

```diff
@@ -97,12 +97,22 @@ func newLibindex(ctx context.Context, indexerCfg config.IndexerConfig, store cci
 		ScanLockRetry:        libindex.DefaultScanLockRetry,
 		LayerScanConcurrency: libindex.DefaultLayerScanConcurrency,
 	}
-
-	opts.ScannerConfig.Repo = map[string]func(interface{}) error{
-		"rhel-repository-scanner": deserializeFunc([]byte(fmt.Sprintf(`{"repo2cpe_mapping_url": "%s"}`, indexerCfg.RepositoryToCPEURL))),
-	}
-	opts.ScannerConfig.Package = map[string]func(interface{}) error{
-		"rhel_containerscanner": deserializeFunc([]byte(fmt.Sprintf(`{"name2repos_mapping_url": "%s"}`, indexerCfg.NameToCPEURL))),
+	err = sanityCheckURL(indexerCfg.RepositoryToCPEURL)
```

Comment:
Let's consider doing this regardless of whether Central/Sensor is available; you might get the file in the sanity check, but something might fail in the actual pull, and things will be stale for 24 hours if I understood the updater logic correctly.

Reply:
Are suggesting we can move this check to somewhere else? Do you have any recommendations?

Reply:
No, I'm suggesting we should not even do the check, but simply use the initial files, always.